CMakeLists.txt

```diff
@@ -45,38 +40,43 @@ option (SINGULARITY_BUILD_STELLARCOLLAPSE2SPINER "Compile stellarcollapse2spiner
 option (SINGULARITY_BETTER_DEBUG_FLAGS
   "Better debug flags for singularity" ON)
 option (SINGULARITY_HIDE_MORE_WARNINGS "hide more warnings" OFF)
-option (SINGULARITY_SUBMODULE_MODE "Submodule mode" OFF)
 option (SINGULARITY_BUILD_CLOSURE "Mixed cell closure" ON)
 option (SINGULARITY_TEST_SESAME "Test the Sesame table readers" OFF)
 option (SINGULARITY_TEST_STELLAR_COLLAPSE "Test the stellar collapse table readers" OFF)
 option (SINGULARITY_TEST_PYTHON "Test the Python bindings" OFF)
 option (SINGULARITY_USE_SINGLE_LOGS "Use single precision logs. Can harm accuracy." OFF)
 option (SINGULARITY_USE_TRUE_LOG_GRIDDING "Use grids that conform to log spacing." OFF)
-option (SINGULARITY_PATCH_MPARK_VARIANT "Apply GPU patch to mpark-variant submodule" ON)
+
+option (SINGULARITY_SUBMODULE_MODE "Submodule mode" OFF)
+option (SINGULARITY_CUSTOM_KOKKOS "Use custom kokkos dir" OFF)
+
+#option (SINGULARITY_PATCH_MPARK_VARIANT "Apply GPU patch to mpark-variant submodule" ON)
 
 # Patches variant to be compatible with cuda
 # Assumes "patch" is present on system
-if(SINGULARITY_PATCH_MPARK_VARIANT)
-  message(STATUS "Patching mpark::variant to support GPUs")
-  execute_process(COMMAND patch -N -s -V never
-                  ${CMAKE_CURRENT_SOURCE_DIR}/utils/variant/include/mpark/variant.hpp
-                  ${CMAKE_CURRENT_SOURCE_DIR}/utils/cuda_compatibility.patch
-                  )
```

Comment:
is this happening elsewhere now?

Reply:
`cmake/singularity-eos/mpark_variant.cmake`